### PR TITLE
feat: preview image

### PR DESCRIPTION
**Pull request by juliahermak**
Screenshot 2024-04-10 00 15 59
Screenshot 2024-04-10 00 16 10

Screenshot 2024-04-10 00 19 17

### CODE REVIEW

**Comment by juliahermak** [comment]
[Feat: Preview image and file](https://trello.com/c/X2LewT0P/528-feat-preview-image-and-file)

File: src/components/AChat/AChat.vue [hunk 1]
```diff
@@ -57,6 +59,10 @@ const emitScroll = throttle(function () {
 
 export default {
   props: {
+    file: {
+      type: [Array, null],
```
Comment (bludnic) [inline src/components/AChat/AChat.vue:63]: remove 
```suggestion
      type: Array,
```

Comment (bludnic) [reply src/components/AChat/AChat.vue:63]: rename the props to `files`

File: src/components/AChat/AChatPreviewFile.vue [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+}
+
+export default defineComponent({
+  components: {
+    IconFile
+  },
+  emits: ['cancel', 'remove-item'],
+  props: {
+    file: {
+      type: [Array, null],
```
Comment (bludnic) [inline src/components/AChat/AChatPreviewFile.vue:64]:
```suggestion
      type: Array,
```

File: src/components/Chat/Chat.vue [hunk 3]
```diff
@@ -496,6 +509,19 @@ export default {
       nextTick(() => this.$refs.chat.scrollToBottom())
       this.replyMessageId = -1
     },
+    handleImageSelected(imageData) {
+      if (this.selectedImage.length < 10) {
```
Comment (bludnic) [inline src/components/Chat/Chat.vue:513]: Move `10` to constants.ts

Comment (bludnic) [reply src/components/Chat/Chat.vue:513]: `export const UPLOAD_MAX_FILE_COUNT = 10`

**Review by bludnic — CHANGES_REQUESTED** [review]

File: src/components/Chat/Chat.vue [hunk 4]
```diff
@@ -496,6 +509,19 @@ export default {
...
+        this.selectedImage.push(imageData)
+      } else {
+        alert('можно загрузить только 10 файлов')
```
[the remark this answers is not in the document]
Comment (bludnic) [reply src/components/Chat/Chat.vue:516]: Instead show a snackbar with the message: You can upload max N files

**Comment by bludnic** [comment]
Closed because of incorrect branch name. Use this PR instead https://github.com/Adamant-im/adamant-im/pull/628

**Pull request closed without merging**